src/app/dashboard/dashboard.component.ts

```diff
@@ -68,7 +65,7 @@ this.router.navigate(['description/',id])
 }
 favFunc(data:any){
   this.service.updateFav(data)
-  .subscribe(d=>{console.log('-----',d);
+  .subscribe(d=>{
```

Comment:
why is d declared..?  its not used at all inside subscribe

Reply:
resolved